src/dosbox.cpp

```diff
@@ -821,12 +821,21 @@ void DOSBOX_Init()
 	pint->Set_help("The OPL waveform is now sampled at the mixer's playback rate to avoid\n"
 	               "resampling.");
 
-	const char* oplmodes[] = {"auto", "cms", "opl2", "dualopl2", "opl3", "opl3gold", "none", nullptr};
+	pstring = secprop->Add_string("cms", when_idle, "auto");
+	pstring->Set_values({"on", "off", "auto"});
+	pstring->Set_help(
+	        "Enable CMS emulation for SoundBlaster 1, 1.5 and 2 ('auto' by default).\n"
+	        "  off:  Disable CMS emulation (except when a GameBlaster is selected).\n"
+	        "  on:   Enable CMS emulation on SoundBlaster 1, 1.5 and 2.\n"
+	        "  auto: Enable CMS emulation for SoundBlaster 1 and GameBlaster.");
```

Comment:
Suggest to go with this:

```
"Enable CMS emulation ('auto' by default).\n"
"  off:   Disable CMS emulation (except when the Game Blaster is selected).\n"
"  on:    Enable CMS emulation on Sound Blaster 1 and 2.\n"
"  auto:  Auto-enable CMS emulation for Sound Blaster 1 and Game Blaster."
```

Main points:

- We don't emulate the Sound Blaster 1.5
- Sound Blaster & Game Blaster (two words)

I will refine the description of `oplmode` shortly myself as it has a lot to be desired 😄

Reply:
Checked that there are no more references to Sound Blaster 1.5. Also corrected the naming of Sound Blaster and Game Blaster in other places (comments only).